packages/langium/src/grammar/lsp/langium-grammar-goto.ts

```diff
@@ -0,0 +1,43 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { isAssignment, isInterface, isParserRule } from '../generated/ast';
+import { DefaultGoToResolverProvider, GoToLink } from '../../lsp';
+import { LangiumServices } from '../../services';
+import { CstNode } from '../../syntax-tree';
+import { getContainerOfType, getDocument } from '../../utils/ast-util';
+
+export class LangiumGrammarGoToResolver extends DefaultGoToResolverProvider {
+
+    constructor(services: LangiumServices) {
+        super(services);
+    }
```

Comment:
Minor: This constructor isn't actually necessary. The constructor of the parent type will be inherited, if nothing else is specified.